src/rating/rating.ts

```diff
@@ -173,9 +175,11 @@ export class NgbRating implements ControlValueAccessor,
     this._updateState(this.rate);
   }
 
+  setDisabledState(isDisabled: boolean) { this.disabled = isDisabled; }
```

Comment:
Yes, this is part of the `ControlValueAccessor` interface and this is what form's infrastructure is going to call. You can see that is needed by trying to remove this line and running unit tests :-)